### PR TITLE
Feature/mane transcript support

### PR DESCRIPTION
**Pull request by aparton**
Adds MANE transcript support to VEP:

- adds --mane flag to add Mane transcript status to output
- Adds --mane to --everything
-Updates -pick and -pick_order to support mane transcripts

PR to public-plugins with associated documentation changes incoming

### CODE REVIEW

**Comment by aparton** [comment]
Documentation updates here: https://github.com/aparton/public-plugins/tree/feature/mane_transcript_support

File: modules/Bio/EnsEMBL/VEP/Config.pm [hunk 1]
```diff
@@ -368,7 +368,7 @@ our %VALID = (
   terms           => [qw(SO display NCBI)],
   sift            => [qw(s p b)],
   polyphen        => [qw(s p b)],
-  pick_order      => [qw(canonical appris tsl biotype ccds rank length ensembl refseq)],
+  pick_order      => [qw(mane canonical appris tsl biotype ccds rank length ensembl refseq)],
```
Comment (ens-lgil) [inline modules/Bio/EnsEMBL/VEP/Config.pm]: Might need to add `mane` to the `%INCOMPATIBLE` hash for `most_severe` and `summary`:
```
our %INCOMPATIBLE = (
most_severe => [qw(no_intergenic protein symbol sift polyphen coding_only ccds mane canonical xref_refseq numbers domains tsl appris uniprot summary pick flag_pick pick_allele flag_pick_allele)],
summary     => [qw(biotype no_intergenic protein symbol sift polyphen coding_only ccds mane canonical xref_refseq numbers domains tsl appris uniprot most_severe pick flag_pick pick_allele flag_pick_allele)],
```

**Comment by aparton** [comment]
> Make sure that the license dates are up to date

The dates on my branch are not correct, but as the PR is against master, then it should be fine and it'll only take the changes?

**Comment by ens-lgil** [comment]
Yes true!

**Pull request merged**